src/app/components/Header/Header.js

```diff
@@ -1,36 +1,57 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import { withRouter } from 'react-router'
 
 import NetworkSwitcher from '../NetworkSwitcher'
 import MainNav from '../MainNav'
 
-import './Header.css'
+import home from '../../../img/home.svg'
 
-const Header = (props) => {
-  const { showMenu, setNetwork, selectedNetworkId, networks } = props
+import style from './Header.css'
+
+const getNavigation = props => {
+  const { account, history } = props
+  const loggedIn = account.address && account.wif
+  const isHomepage = history.location.pathname === '/'
```

Comment:
This type of logic shouldn't generally be inside of a component because it's pulling in app state into something that should be independent.

Reply:
Maybe the container including this container can include this based on a prop ... like showNav ...